gst/nnstreamer/tensor_if/gsttensorif.h

```diff
@@ -140,6 +166,8 @@ struct _GstTensorIf
   GList *cv_option;
   GList *then_option;
   GList *else_option;
+
+  custom_cb_s *custom;
```

Comment:
custom_cb_s has a fixed size. Let's not use pointer here.

Reply:
Thank you. Updated.